CHANGES.md

```diff
@@ -76,7 +76,10 @@
 
 ## Breaking Changes
 
-* X behavior was changed ([#X](https://github.com/apache/beam/issues/X)).
+* X behavior was changed ([#X](https://github.com/apache/beam/issues/X)).=======
+* Yapf version upgraded to 0.43.0 for formatting (Python) ([#34801](https://github.com/apache/beam/pull/34801/)).
+* Extra packages moved out of `install_required` to individual extra sections (`hdfs`, `mongodb`, `redis`) as part of [#35297](https://github.com/apache/beam/pull/35297)
```

Comment:
Could we mention that the others have been moved to separate (existing) extras as well?

Reply:
Good point.